examples/pytorch/elastic/imagenet/Dockerfile

```diff
@@ -1,10 +1,10 @@
-ARG BASE_IMAGE=pytorch/pytorch:1.13.1-cuda11.6-cudnn8-runtime
+ARG BASE_IMAGE=pytorch/pytorch:2.2.1-cuda12.1-cudnn8-runtime
```

Comment:
Should we use the same image as for Katib in this PR: https://github.com/kubeflow/katib/pull/2279.
```
nvcr.io/nvidia/pytorch:24.01-py3
```
To support ARM-based builds in the future?